x-pack/legacy/plugins/lens/public/indexpattern_plugin/dimension_panel/format_selector.tsx

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiFormRow, EuiFieldNumber, EuiComboBox } from '@elastic/eui';
+import { KBN_FIELD_TYPES, ES_FIELD_TYPES } from 'src/plugins/data/common';
+import { IndexPattern } from '../types';
+import { IndexPatternColumn } from '../indexpattern';
+import { IndexPatternDimensionPanelProps } from './dimension_panel';
+
+const supportedFormats: Record<string, { title: string }> = {
+  number: {
+    title: i18n.translate('xpack.lens.indexPattern.numberFormatLabel', {
+      defaultMessage: 'Custom number',
```

Comment:
```suggestion
      defaultMessage: 'Decimal',
```

Reply:
Went with "Number" now that we have a "Default" setting